day-4.lisp

```diff
@@ -0,0 +1,32 @@
+(defpackage aoc-day-4
+  (:use :cl)
+  (:export :answers))
+(in-package :aoc-day-4)
+
+(ql:quickload "md5")
```

Comment:
This appears to be quicklisp, a dep manager of sorts. TIL

Reply:
Yup that's exactly right.